### DISCUSSION TITLE
Javascript import export error

### DISCUSSION THREAD

**Discussion by legendSabbir** [mkslanc/ace-linters#58] [General]
version 0.10.0

import export showing error in javascript . And there in error code .

```javascript
import abc from "./abc.js"

// error
[
    {
        "row": 0,
        "column": 0,
        "text": "Parsing error: 'import' and 'export' may appear only with 'sourceType: module'",
        "type": "error"
    }
]
```

how can i provide sourceType to module ? any option available ?

**Comment by mkslanc** [comment]
```javascript
languageProvider.setGlobalOptions("javascript", {
    parserOptions: {
        "sourceType": "module"
    }
});
```

**Reply by legendSabbir** [reply]
a documentation for `ace-linters` will be good

**Reply by legendSabbir** [reply]
Ace linters shouldn't overwrite autocomplete menu for modes that it doesn't have services . For example when ace linters is registered if i switch mode from `javascript` to `bash` (any mode without services)  default autocomplete suggestions doesn't show up because it is overwritten by ace linters .
Whats you thought on this ?

**Reply by mkslanc** [reply]
Yeah, I agree, but is it overwriting with 
```javascript
let options = {functionality: completion: {overwriteCompleters: false}}
```
?

**Reply by legendSabbir** [reply]
no it is not overwriting with this . But using `options` will also give unnecessary suggestions .
If anyone want to allow custom suggestions they can do that now using the `feature` option in worker

edit: my bad feature `completion: false`totally removes completers
